.travis.yml

```diff
@@ -9,6 +9,7 @@ env:
   - KAFKA_PEERS=localhost:6667,localhost:6668,localhost:6669,localhost:6670,localhost:6671
   - KAFKA_INSTALL_ROOT=/home/travis/kafka
   - KAFKA_HOSTNAME=localhost
+  - DEBUG=true
```

Comment:
You may need to change this to LOGGER=true to make it work; I think DEBUG is reserved by Travis or something.